src/org/labkey/test/TestScrubber.java

```diff
@@ -213,4 +222,13 @@ private void disableFileUploadSetting() throws IOException, CommandException
             }
         }
     }
+
+    private void clearListOfAllowedFileExtensions() throws IOException, CommandException
```

Comment:
Can we consolidate this with the copy in `BaseWebDriverTest`?

Reply:
I've removed this method. The only place this code should now appear is in `AllowedFileExtensionAdminPage.deleteAllAllowedFileExtension`.